rest-api/src/main/resources/definitions/study.yml

```diff
@@ -27,6 +27,17 @@ properties:
         description: |
             Client data for a study should be in a syntactically valid JSON format. It will be returned as is to the client (as JSON). It can contain any information that is useful to study authoring tools or administration tools that is not part of the Bridge data model.
         additionalProperties: true
+    studyTimeZone:
+        type: string
+        example: 'America/Los_Angeles'
```

Comment:
document the default using default; make sure the object browser shows default values on developer.sagebridge.org.

Reply:
Updated